_data/team.yml

```diff
@@ -91,14 +84,3 @@
       - bioinformatics
       - excercise
       - microbiome
-
-- name: Max von Hippel
```

Comment:
Can you move Max back to the end? I think we were alphabetizing on "von" instead of "Hippel" (I don't have strong feelings either way, but probably best to keep the ordering how it was)